### DISCUSSION TITLE
現代人效率至上的生活真的值得嗎？

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2187] [General]
唸期中唸到一半有感而發，最近發現自己有點過於追求效率，總是忙著完成各種任務，卻很少停下來思考生活的意義。
努力工作上課不就是為了體驗生活的美好嗎。但最近發現過著汲汲營營的生活，自己真的錯過了很多生活中的美好風景。
我開始思考，這樣不停汲汲營營地追求效率，到底是為了什麼？
或許生活應該慢一點，去體會那些真正讓人快樂的瞬間，而不是一味地提高生產力...

**Comment by Mminniehuang** [comment]
突然想到昨天聽到一句話，我們一生所追求的最終目標就是幸福。
（不過可能記得不太準確，也可能沒有完全體會演講者所要傳達的理念和深度，單純分享！！

這讓我想了很多很多，現在的努力跟追求不外乎是為了以後能有更好的經濟條件和生活品質，進而獲得幸福。效率至上確實可以讓我們更快更好的達到，但是我真的好不快樂。
我一直在想或許我應該有效率的解決該做的，然後花更多時間尋找自己尋找快樂。

**Comment by wcyeee** [comment]
因為我個人很喜歡有效率的生活，所以我覺得追求效率對我來說是一件會讓我快樂的事情耶！ 如果今天我覺得自己很有效率的話，我會覺得我今天過的很充實；反而如果今天沒什麼效率，就是很輕鬆的做了一些些事情而已，反而會讓我覺得很焦慮。  但到頭來我覺得你想表達的是追求效率可能會錯過很多生活中的美好，這點我也是認同的！雖然我是從追求效率中得到成就感和快樂，但確實有時候做事情做的太快或太急也會對我產生一些負面影響（也許我追求效率快樂的來源是來自有效率的完成某件事之後的成就感？）所以認真享受生活的美好確實是大部分人現在一個很重要的議題。

**Comment by ShuliANN** [comment]
我認為追求效率本身是好的。若我們能夠有效率地完成必須做的事，那麼剩下的時間可以用來進行更多我們真正熱愛的活動。效率高並不與生活品質相衝突，反而能讓我們在有限的時間內有更多的選擇空間去享受生活、探索興趣，甚至是休息和放鬆。因此，效率至上的生活方式，其實是讓我們能夠更靈活地掌握時間，提升生活的豐富性。

**Comment by lai1008** [comment]
我認為追求效率本身是好的。因為剩下的時間可以用來進行更多我們真正熱愛的活動。效率高並不與生活品質相衝突，反而能讓我們在有限的時間內有更多的選擇空間去享受生活、探索興趣，甚至是休息和放鬆。

**Comment by 112121py** [comment]
我覺得有效率是件好事，應該說我們要調整日常的生活節奏，該讀書該做報告該忙的時候就應該要有效率的執行，那麼因為「有效率」而省下來的時間，就可以拿來好好的放鬆、休息、做自己想做的事，讓自己的節奏慢下來，這樣有快有慢的生活才不至於讓自己太疲乏或無趣

**Comment by Taoyi-Chiang** [comment]
我覺得完全不值得耶，畢竟人類生來並不是只是為了完成一個個的任務和工作。
就像原PO說的，我覺得人應該把生活的焦點放在體驗當下的感受，而不是把寶貴的注意力放在工作上。
之前看到有個說法是，在公司不要表現的能力太好，因為能者過勞，只會丟過來越來越多的工作，但不會反映在薪水上。
我覺得滿有道理的，反正期限內該交的東西都有交，對我來說這樣就可以了。
多出來的時間發呆睡覺滑手機說不定都比較賺？

**Comment by sRain0113** [comment]
追求效率對我來說是件快樂的事，當寫日記時回想起今天完成了多少事，心情會覺得特別愉悅，並且期許自己明天也能過的跟今天一樣充實。不過當我覺得體力或是精神其中一項快跟不上時，還是會適時的放鬆休息，不然再繼續做事，效率也是高不了的。我要趁大學時期把想學的東西、想做的事都試過一遍，以免之後想做卻沒時間而感到後悔。

**Comment by DarsooK** [comment]
現代社會追求效率至上的生活方式確實帶來了很多便利和成就，但這種生活方式也引發了許多值得深思的問題。高效率不斷推動人們在工作、生活中追求更快的速度和更大的產出，但同時也可能讓人忽視一些重要的價值，比如健康、內在成長和人際關係。我認為：這種生活方式是否真的「值得」，取決於我們如何看待生活的意義和平衡。

**Comment by gigitilto** [comment]
你說的不無道理，但我認為追求效率是非常重要的，因為提高了效率，就可以省下時間做喜歡做的事，若像你所說的提高效率無窮無盡地完成該做的事，那我覺得這樣事情的安排就不是有效率的，反而失去了追求效率的本質了。

**Comment by forestdew507** [comment]
雖然我也不是很有經驗，但目前體驗了 20 年來，覺得忙碌的盡頭就是沒有所謂輕重緩急，只有緩急XD 在段考期間，我覺得追求效率至上沒什麼不好，也不必刻意花時間思考意義 ( 不過當然了~ 如果反射出現的靈感還是可以去體會啦XD 畢竟 fu 來了錯過就不再 )。

btw 但是我覺得內心繼續看著自己的初衷是很重要的，所謂初衷和意義多少有重疊，不過這裡指得初衷是大方向的初心而非細究 ( 例如前幾天跟一個朋友聊天聊到他北上讀書的一大原因是為了北部的豐富藝文資源，但忙一忙就被扔到背後，這個時候正好是期中周，這樣的初衷我想勢必會讓他開始為未來種下一些種子，或者說在這波忙碌過後就會回頭填充自己的生活意義了:) )

**Comment by yuan3x2** [comment]
我反而認為有效率的完成工作能夠增加我的成就感，也減少了我的焦慮，不需要時時擔心著是否還有未完成的任務，此外，在完成任務的過程中，某方面來說也算是在為自己創造價值，能夠在短時間內達到效益，將省下來的時間花在自己的興趣愛好，相比慢慢悠悠更能讓我感到快樂。

**Comment by yue083139** [comment]
我覺得效率的完成工作是為了讓我能夠好好的享受剩下來的生活，不用每次在打電動或是出去玩的時候還要時時刻刻想著有哪些工作沒完成，反而沒有放鬆到，因此我認為效率的生活並沒有甚麼不好的地方，只是就算沒有達到預期的目標，也不要給自己太大的壓力反而本末倒置。
